acceptor/settings.py

```diff
@@ -38,8 +38,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'analytics.apps.AnalyticsConfig',
-    'charts.apps.ChartsConfig',
+    'edx_global_analytics.analytics.apps.AnalyticsConfig',
+    'edx_global_analytics.charts.apps.ChartsConfig',
```

Comment:
What about `olga.analytics.apps.AnalyticsConfig`?
I'd like to keep names short, but reasonable